admin/preprints/views.py

```diff
@@ -558,7 +560,10 @@ class PreprintMakePublishedView(PreprintMixin, View):
 
     def post(self, request, *args, **kwargs):
         preprint = self.get_object()
-        preprint.set_published(True, request, True)
+        preprint.set_published(True, request, True, True)
```

Comment:
That's a lot of unlabeled args there. Could you replace those with kwargs so it's more clear what's going on?